pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -132,29 +135,134 @@ masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask')).show(10)
 
-Quick demo of one ND tile one raw tile
+```python
+display(collected[0].tile_nd_1)
+```
 
-Quick demo of ND in two different cell types
+And the tile named `tile_nd_2` has the values of 1 and 2 masked out. This is because we created the tile by setting a new user defined NoData value to `tile_nd_1` the values previously masked out in `tile_nd_1` stayed masked when creating `tile_nd_2`.
+
+```python
+display(collected[0].tile_nd_2)
+```
+
+## Nodata Values in Aggregation
+
+Let's use the same tile as before to demonstrate how NoData values affect tile aggregations
+
+```python
+tile_size = 100
+x = np.zeros((tile_size, tile_size), dtype='int16')
+for i in range(4):
+    x[:, i*tile_size//4:(i+1)*tile_size//4] = i
+x = Tile(x)
+
+rf = spark.createDataFrame([Row(tile=x)])
+display(x)
+```
+
+First we create the two new masked tile columns as before. One with only the value of 1 masked, and the other with and values of 1 and 2 masked. 
+
+```python
+def get_nodata_ct(nd_val):
+    return CellType('uint16').with_no_data_value(nd_val).cell_type_name
+
+masked_rf = rf.withColumn('tile_nd_1', 
+                           rf_convert_cell_type('tile', get_nodata_ct(1))) \
+              .withColumn('tile_nd_2',
+                          rf_convert_cell_type('tile_nd_1', get_nodata_ct(2)))
+```
 
+The results of `rf_tile_sum` vary on the tiles that were masked. This is because any cells with NoData values are ignored in the aggregation. Note that `tile_nd_2` has the lowest sum, since it has the fewest amount of data cells.
 
+```python
+masked_rf.select(rf_tile_sum('tile'), rf_tile_sum('tile_nd_1'), rf_tile_sum('tile_nd_2')).show()
+```
+
+
+## Combining Tiles with Different Data Types
+
+RasterFrames supports having Tile columns with multiple cell types in a single RasterFrame. It is important to understand how these different cell types interact. 
+
+Let's first create a RasterFrame that has columns of `float` and `int` cell type. 
+
+```python
+x = Tile((np.ones((100, 100))*2).astype('float'))
+y = Tile((np.ones((100, 100))*3.0).astype('int32'))
+rf = spark.createDataFrame([Row(x=x, y=y)])
+
+rf.select(rf_cell_type('y'), rf_cell_type('x')).distinct().show()
+```
+
+When performing a local operation between tile columns with cell types `int` and type `float`, the resulting tile cell type will be `float`. If two tiles of different sized integer cell types are added together in the same way, the resulting cell type will be the same as the largest of the two integer cell types.
+
+```python
+rf_added = rf.withColumn('xy_sum', rf_local_add('y', 'x'))
+rf_added.select(rf_cell_type('xy_sum'), rf_cell_type('y'), rf_cell_type('x')).distinct().show()
+```
+
+Combining tile columns of different cell types gets a little trickier when user defined NoData cell types are involved. Let's create 3 tile columns: one without a defined NoData value, one with a NoData value of 1, and one with a NoData value of 2.
```

Comment:
the first tile you should say one with the "default NoData value of zero" versus wihtout defined NoData. It's the distinction between `uint16` (has nodata, nodata value is 0) and `uint16raw` (does not have a nodata) .

Reply:
My understanding is the `int16` does not have a NoData value of 0, while `uint16` does

Reply:
Oh but that function makes tiles with `uint16` nvm

Reply:
@vpipkt I changed this section to remove the confusing between uint16 and int16.